samples/basic/blinky/boards/intel_socfpga_agilex_socdk.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	aliases {
+		led0 = &myled0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		myled0: led_0 {
+			gpios = <&gpio1 19 GPIO_ACTIVE_HIGH>;
+		};
+	};
+};
+
+&gpio1 {
+	status = "okay";
+};
```

Comment:
Why not put these in the board devicetree?

Reply:
We are currently try to enable as minimum as possible in board dtsi file. GPIO is not an essential peripheral for our platform bring up. So by default all non-essential peripheral are turned off. 
Also our pins could be re-configured for various other purpose like SPI, I2C, SDMMC , etc, Depending on end application use case, respective peripherals could be enabled flexibly. For example as this is blinky application, we can enable only GPIO peripheral here with required alias for sample application.